pyproject.toml

```diff
@@ -160,7 +160,7 @@ wcslint = "astropy.wcs.wcslint:main"
 
 [build-system]
 requires = ["setuptools",
-            "setuptools_scm>=6.2",
+            "setuptools_scm>=8.0.0",
```

Comment:
Since you want to backport, I wouldn't bump the version here. Or we can choose not to backport, especially if CI is not affected.

Reply:
The bump is required for the rest of the patch to function, but I guess not backporting is fine !